### DISCUSSION TITLE
Update the "Task" summary at the top of the chat

### DISCUSSION THREAD

**Discussion by orangejon** [cline/cline#1078] [Feature Requests]
I type a task, Cline generates a bunch of output writing and testing code, I type another task, etc...

However, with so much output in the chat it's hard to remember what was the last prompt I gave it. (did I type it, or just think about it?)

I think it would be **much** more useful for the "Task" snippet at the top of the chat showed the last task prompt I submitted, not the first one in this chat. (At least while you have that chat 'open' - I guess the 'index' page could still show the first prompt)

**Comment by steveterryp** [comment]
I'd like to be able to edit the Task banner, so I can refer to it later, knowing what I did in the chat
